Content.Server/GameObjects/Components/Research/ResearchServerComponent.cs

```diff
@@ -160,12 +172,9 @@ public void UnregisterClient(ResearchClientComponent client)
             Clients.Remove(client);
         }
 
-        public void Update(float frameTime)
+        private void OnResearchPointGet()
         {
-            if (!CanRun) return;
-            _timer += frameTime;
-            if (_timer < 1f) return;
-            _timer = 0f;
+            if (!CanRun || _pauseManager.IsEntityPaused(Owner)) return;
```

Comment:
This still isn't correct because the timer will keep running. This means that if you manage to pause at the perfect time you could advance the game while having research points get stuck.